data/src/main/java/io/gnosis/data/backend/dto/TransactionDto.kt

```diff
@@ -1,16 +1,60 @@
 package io.gnosis.data.backend.dto
 
+import pm.gnosis.model.Solidity
+
 data class DataDecodedDto(
     val method: String,
-    val parameters: List<ParamsDto>?
+    val parameters: List<ParamDto>?
 )
 
-data class ParamsDto(
-    val name: String,
-    val type: String,
-    val value: Any
+data class ValueDecodedDto(
+    val operation: Operation,
+    val to: Solidity.Address,
+    val value: Long,
+    val data: String,
+    val dataDecoded: DataDecodedDto?
 )
 
+sealed class ParamDto {
```

Comment:
Would maybe be better to have a type parameter for `value` that is later defined foreach concretisation. Although, it is either `String`, `List<String>` or `Address` as far as I could see.

Reply:
for v1 it should be fine I think. And we only deal with Strings for ui purposes